### PR TITLE
Implementation BidList + logging user + Spring Boot security

### PR DESCRIPTION
**Pull request by ALEXD2758**
Implementation of BidListController/Interface/Service/Model
Implementation logging user and Spring Boot  security (part of Story_7)

### CODE REVIEW

File: src/main/java/com/alex/poseidon/controllers/BidListController.java [hunk 1]
```diff
@@ -1,55 +1,92 @@
...
 
 @Controller
-public class BidListController {
-    // TODO: Inject Bid service
+public class BidListController implements BidListControllerInterface {
 
+    @Autowired
+    BidListService bidListService;
+
+    @Override
```
Comment (akath19) [inline src/main/java/com/alex/poseidon/controllers/BidListController.java:33]: Each method in each controller needs to have JavaDoc for it

**Review by akath19 — CHANGES_REQUESTED** [review]
Please add JavaDoc to all methods in all controllers

File: src/main/java/com/alex/poseidon/controllers/BidListController.java (hunk 1 above)
Comment (ALEXD2758) [reply src/main/java/com/alex/poseidon/controllers/BidListController.java:33]: Definitive JavaDoc implemented for BidListController/HomeController/LoginController
Temporary JavaDoc implemented for CurveController/RatingController/RuleNameController/TradeController, to go through later in each story.

**Comment by akath19** [comment]
lgtm

**Review by akath19 — APPROVED** [review]

**Pull request merged**